aggregation-worker/aggregate.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# mount efs volume 
+
+# run aggregator
+java -jar /opt/aggregator/*.jar $*
```

Comment:
As discussed would also be good perhaps to dispense with the bash layer and use the copy dependencies example shown in the dockerfile-maven integration tests to add dependencies as a separate layer to speed up builds/deployment